website/core/Footer.js

```diff
@@ -51,11 +51,11 @@ class Footer extends React.Component {
               User Showcase
             </a>
             <a
-              href="http://stackoverflow.com/questions/tagged/"
+              href="http://stackoverflow.com/questions/tagged/unibeautify"
               target="_blank">
               Stack Overflow
             </a>
-            <a href="https://discordapp.com/">Project Chat</a>
+            <a href="#">Project Chat</a>
```

Comment:
Should we have a Gitter or public Slack or something?

Reply:
I think so.  Do you want to use the Slack channel we have or something else?  Mainly just wanted the link to Discord out of there as it wasn't relevant.

Reply:
Slack may be a good idea. Use https://github.com/rauchg/slackin ?

Reply:
Heroku seems to be the most popular.